repo/packages/S/spark/2/config.json

```diff
@@ -0,0 +1,86 @@
+{
+  "type": "object",
+  "properties": {
+    "mesos": {
+      "description": "Mesos specific configuration properties",
+      "type": "object",
+      "properties": {
+        "master": {
+          "default": "mesos://zk://master.mesos:2181/mesos",
+          "description": "The URL of the Mesos master. The format is a comma-delimited list of hosts like zk://host1:port,host2:port/mesos. If using ZooKeeper, pay particular attention to the leading zk:// and trailing /mesos! If not using ZooKeeper, standard host:port patterns, like localhost:5050 or 10.0.0.5:5050,10.0.0.6:5050, are also acceptable.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "master"
+      ]
+    },
+    "spark": {
+      "description": "Spark Framework Configuration Properties",
+      "type": "object",
+      "properties": {
+        "framework-name": {
+            "description": "The name of the framework. Until this is configurable, please do not change this from it's default value.",
+            "type": "string",
+            "default": "spark"
+        },
+        "host": {
+          "type": "string",
+          "description": "The host that the Spark cluster dispatcher will register under.",
+          "default": "$HOST"
+        },
+        "cpus": {
+          "default": 1,
+          "description": "CPU shares to allocate to each Marathon instance.",
+          "minimum": 0.0,
+          "type": "number"
+        },
+        "instances": {
+          "default": 1,
+          "description": "Number of Marathon instances to run.",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "mem": {
+          "default": 1024.0,
+          "description": "Memory (MB) to allocate to each Marathon task.",
+          "minimum": 1024.0,
+          "type": "number"
+        },
+        "mesos-authentication-principal": {
+          "description": "The Mesos principal used for authentication.",
+          "type": "string"
+        },
+        "mesos-authentication-secret-file": {
```

Comment:
Yes, it might be helpful to double check with @kensipe. Or if you know how these command line params are used, you can resolve the issue yourself by updating the marathon.json. We would like to avoid including unused options.